src/main/resources/db/changelog/changelog-1.yaml

```diff
@@ -0,0 +1,59 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: Victor
+      preConditions:
+        onFail: MARK_RAN
+        not:
+          tableExists:
+            tableName: users
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: email
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: date_of_birth
+                  type: TIMESTAMP
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: 2
+      author: Victor
+      changes:
+        - insert:
```

Comment:
de ce ai folosit acest change set?

Reply:
nu stiu la ce te referi exact aici, dar am citit ca de fiecare data cand faci o schimbare este bine sa o faci separat ca sa fie mai simplu sa faca roll back in caz de ceva nu merge cum trebuie, adica sa facem de fiecare data un alt changeSet care este tratat individual